File: Scripts/volatile_and_force_compiler_task/classes/python/classes.py
class MyClass:

   def __init__(self,  variable):
      self.variable = variable
   
   def displayMyClass(self, i):
	self.variable = 1 + i

def executeTask(i):
	obj = MyClass( 2000)
	obj.displayMyClass(i)
	return i

r = 1

for i in xrange(2100000000):
	r = executeTask(i + r)

File: scripts/volatile_and_force_compiler_task/classes/perl/classes.pl
package SampleObject;

use strict;
use warnings;

sub new {
    my ($class, %args) = @_;
    return bless { %args }, $class;
}

sub sample_method {
    my ($self, $i) = @_;
    $i = 1 + $i;
}

sub executeTask {
	my ($i) = @_;
	my $obj = SampleObject->new( sample_data => 'Variable value is ',
                             more_data   => "1\n" );
	$obj->sample_method($i);
	return ($i );
}
my $r = 1;

for( $a=0; $a < 2100000000; $a = $a + 1) {
	$r = executeTask($a + $r);
}

